utils/OZTestUtil.js

```diff
@@ -49,7 +48,11 @@ const {
   entry,
 } = require("./DeploymentUtil");
 
+const { expectRevert } = require("@openzeppelin/test-helpers");
+const { expect } = require("chai");
+
 const deployFunction = async (contractInterface, args, from) => {
+  if (!contractInterface) return null;
```

Comment:
did you see a case where we had ac contractInterface null and we did not want an error?
I think that contractInterface not defined is a signed of an error or issue and it's best to fail fast IMO

Reply:
Agreed. I will update that.